bin/tronview

```diff
@@ -95,16 +96,26 @@ def view_job(args, job_id, client):
 
 
 def view_job_run(args, job_run_id, client):
-    actions = client.job_runs(job_run_id.url)
+    try:
+        actions = client.job_runs(job_run_id.url)
+    except RequestError as e:
+        response = e.args[1]
```

Comment:
Now I understand. I've never seen this pattern before for moving objects between exceptions.

I don't know man, can you... not do this and instead only pass a string and do whatever parsing you need to do at exception handling time?